src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -191,10 +192,38 @@ public void Write(JsonWriter writer)
                 writer.WritePropertyName("tags");
                 writer.WriteStartObject();
 
-                foreach (var tag in this.Tags.Value)
+                // this will be used when we convert int, double, int[], double[] to string
```

Comment:
This looks familiar, when we did it before was it in a directive for netcoreapp or netstandard or something?

Reply:
Yes. The same file, but for net46, standard that can use system.text.json

Reply:
To facilitate, i pushed a new version where i create a helper function. This function will be used in all places to convert an object to string based on its type.